### DISCUSSION TITLE
What is ms-sdk from TikTok?/Why are the ms-sdk domains blocked?

### DISCUSSION THREAD

**Discussion by maxoux53** [badmojr/1Hosts#929] [General]
Why is the ms-sdk from TikTok blocked by the list? What is it?

![image](https://user-images.githubusercontent.com/71935156/209967546-15033709-107c-488b-9782-7512995622dd.png)
![image](https://user-images.githubusercontent.com/71935156/209967571-d5f57acd-fc67-46b9-ad6b-e677edaba75a.png)

**Comment by SysAdminDoc** [comment]
Old post, but mssdk tracks installs, what you add to cart, and purchases. Blocking it breaks nothing afaik.
